### DISCUSSION TITLE
Feedback Hub

### DISCUSSION THREAD

**Discussion by mateuszmigas** [mateuszmigas/painting-droid#56] [Ideas]
👋 Welcome to Feedback Hub for Painting Droid, here you can share ideas for new features or improvements 👋

**Comment by mateuszmigas** [comment]
**Rename workspace dialog**

**Reply by Kuchasz** [reply]
I'd consider adding workspace name input field to the workspace creation dialog. Some autogenerated default name would be welcome. Not sure about name changing using dialog. Maybe workspace resize & rename should be possible on the same view?

**Reply by mateuszmigas** [reply]
https://github.com/mateuszmigas/painting-droid/issues/108

**Comment by mateuszmigas** [comment]
**Layer properties dialog**

Allow to edit name/opacity/channels

**Reply by mateuszmigas** [reply]
https://github.com/mateuszmigas/painting-droid/issues/108

**Comment by mateuszmigas** [comment]
**Share option**

Allow users to share current workspace. In browser it could be https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share and on desktop something similar.

**Comment by mateuszmigas** [comment]
**Save as dialog**

User needs a way to configure quality of the image and other stuff when saving jpg/png files.
